hax/treemakers/corrections.py

```diff
@@ -11,7 +11,8 @@ class Corrections(TreeMaker):
     Provides:
     - Corrected S1 contains xyz-correction:
       - cs1: The corrected area in pe of the main interaction's S1 using NN 3D FDC
-      - cs1_tpf_2dfdc: Same but for TPF 2D FDC
+      - cs1_true: Same but for after correction of electric field effects
+      - cs1_tpf_2dfdc: Same as cs1 but for TPF 2D FDC
```

Comment:
Unused? Remove?

Reply:
I'm not sure if actually someone uses that variables 'cs1_tpf_2dfdc' still for comparison or some other analysis. If it is the case probably it could be removed.